src/main/java/de/tum/in/www1/artemis/service/ComplaintService.java

```diff
@@ -110,7 +110,7 @@ else if (complaint.getComplaintType() == ComplaintType.MORE_FEEDBACK && !course.
     }
 
     public Optional<Complaint> getByResultId(long resultId) {
```

Comment:
we should remove such methods in services if they include one liners that delegate to the repository.
Then the caller could also use the repository directly (open layered architecture)